### DISCUSSION TITLE
Possible OpenVPNAdmin name change

### DISCUSSION THREAD

**Discussion by bnhf** [bnhf/openvpn-admin-plus#16] [Q&A] [answered]
@karabelnikov 

I'm considering a minor name change to the "OpenVPNAdmin" used in the upper left of each web page in the UI. I'd like to reflect the updated nature of this project, given the original version remains as it was in 2017.

Under consideration:

OpenVPNAdmin+
OpenVPNAdmin Plus
OpenVPNAdmin 2.5+

I'm leaning towards the last one, as the original version of the project works with OpenVPN 2.4 and earlier, where this version requires 2.5 or higher. What do you think? Once decided, I'm planning a write-up using my Wordpress site:

https://technologydragonslayer.com

**Comment by karabelnikov** [comment, marked as answer]
**@bnhf** 

Scott, changing the project name is a good idea, but I think linking the project name to the OpenVPN version is not entirely correct. The name "OpenVPN Admin Plus" looks good, it reflects in the name, as it were, an improved version of the project. You can think about this option :)
